Updater/EvaluateBronoldFehskeBCImpl.cpp

```diff
@@ -0,0 +1,97 @@
+#include <EvaluateBronoldFehskeBCImpl.h>
+#include <math.h>
```

Comment:
Our standard has been to instead use <cmath> and use std::sqrt and std::pow, etc.